versions/v1/units.json

```
@@ -1642,5 +1642,842 @@
+        ],
+        "quantity": "Power Per Area",
+        "conversion": {
+            "multiplier": 0.115740740740741,
+            "offset": 0.0
+        },
+        "source": "qudt.org",
+        "sourceReference": "http://qudt.org/vocab/unit/J-PER-CentiM2-DAY"
+    },
+    {
+        "externalId": "power_per_area:w-per-in2",
+        "name": "W/in\u00b2",
+        "longName": "Watt per Square Inch",
+        "aliasNames": [
+            "W/in\u00b2",
+            "Watt per Square Inch"
+        ],
+        "quantity": "Power Per Area",
+        "conversion": {
+            "multiplier": 1550.0031,
+            "offset": 0.0
+        },
+        "source": "qudt.org",
+        "sourceReference": "http://qudt.org/vocab/unit/W-PER-IN2"
+    },
+    {
+        "externalId": "power_per_area:ft-lb_f-per-ft2-sec",
+        "name": "ft\u22c5lbf/ft\u00b2s",
+        "longName": "Foot Pound Force per Square Foot Second",
+        "aliasNames": [
+            "ft\u22c5lbf/ft\u00b2s",
+            "Foot Pound Force per Square Foot Second"
+        ],
+        "quantity": "Power Per Area",
+        "conversion": {
+            "multiplier": 14.5939042,
+            "offset": 0.0
+        },
+        "source": "qudt.org",
+        "sourceReference": "http://qudt.org/vocab/unit/FT-LB_F-PER-FT2-SEC"
+    },
+    {
+        "externalId": "power_per_area:w-per-m2",
+        "name": "W/m\u00b2",
+        "longName": "watt per square metre",
+        "aliasNames": [
+            "W/m\u00b2",
+            "W/m2",
+            "watt per square metre"
+        ],
+        "quantity": "Power Per Area",
+        "conversion": {
+            "multiplier": 1.0,
+            "offset": 0.0
+        },
+        "source": "qudt.org",
+        "sourceReference": "http://qudt.org/vocab/unit/W-PER-M2"
+    },
+    {
+        "externalId": "power_per_area:w-per-centim2",
+        "name": "W/cm\u00b2",
+        "longName": "Watt per Square Centimetre",
+        "aliasNames": [
+            "W/cm\u00b2",
+            "W/cm2",
+            "Watt per Square Centimetre"
+        ],
+        "quantity": "Power Per Area",
+        "conversion": {
+            "multiplier": 10000.0,
+            "offset": 0.0
+        },
+        "source": "qudt.org",
+        "sourceReference": "http://qudt.org/vocab/unit/W-PER-CentiM2"
+    },
+    {
+        "externalId": "power_per_area:btu_it-per-sec-ft2",
+        "name": "Btu{IT}/(s\u22c5ft\u00b2)",
+        "longName": "BTU per Second Square Foot",
+        "aliasNames": [
+            "Btu{IT}/(s\u22c5ft\u00b2)",
+            "BTU per Second Square Foot"
+        ],
+        "quantity": "Power Per Area",
+        "conversion": {
+            "multiplier": 11356.5267,
+            "offset": 0.0
+        },
+        "source": "qudt.org",
+        "sourceReference": "http://qudt.org/vocab/unit/BTU_IT-PER-SEC-FT2"
+    },
+    {
+        "externalId": "power_per_area:w-per-ft2",
+        "name": "W/ft\u00b2",
```

```suggestion
        "name": "W/ft²",
```
for the characters that are allowed in JSON, could we just use the non-escaped form?

---

we should be able to use `json.dumps(json.loads(test), ensure_ascii=False)` in python to achieve this
(if you are using python for this?)

---

Sure, not all characters support this, but we can try to adhere to this pattern when possible

---

Done, your suggestion worked like a charm!